src/System.Private.CoreLib/src/System/Runtime/Loader/AssemblyLoadContext.CoreCLR.cs

```diff
@@ -52,6 +52,34 @@ internal unsafe Assembly InternalLoad(ReadOnlySpan<byte> arrAssembly, ReadOnlySp
 
             return loadedAssembly;
         }
+        
+#if !FEATURE_PAL
```

Comment:
Why is this related to `FEATURE_PAL`? I think @elinor-fung added `PLATFORM_WINDOWS` which seems more appropriate.

Throughout.

Reply:
It's related to `FEATURE_PAL` because the underlying function in the runtime,  `PEImage::LoadImage(HMODULE)`, is ifdef'd out based on `FEATURE_PAL`. Looking further down the stack, there at the `RawImageLayout` constructor, it looks like the `bTakeOwnership` flag is unimplemented when using the PAL. And then the usage in `GetIJWHostForModule` is for consistency.
https://github.com/dotnet/coreclr/blob/8497ae3ed438f443b22b00a66f42afe8f6623bbf/src/vm/peimagelayout.cpp#L365